contracts/LinearPremiumPriceOracle.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity >=0.5.0;
+
+import "./SafeMath.sol";
+import "./StablePriceOracle.sol";
+
+contract LinearPremiumPriceOracle is StablePriceOracle {
+    using SafeMath for *;
+
+    uint GRACE_PERIOD = 90 days;
+
+    uint public initialPremium;
```

Comment:
How I can access to initialPremium? You probably need a wrapper function like ETHRegistrarController.rentPrice, or make  ETHRegistrarController.prices variable public

Reply:
See above.